website/docs/r/autoscaling_group.html.markdown

```diff
@@ -374,7 +397,9 @@ Note that if you suspend either the `Launch` or `Terminate` process types, it ca
 * `instance_refresh` - (Optional) If this block is configured, start an
    [Instance Refresh](https://docs.aws.amazon.com/autoscaling/ec2/userguide/asg-instance-refresh.html)
    when this Auto Scaling Group is updated. Defined [below](#instance_refresh).
-
+* `warm_pool` - (Optional)  if this block is configured, add a [Warm Pool](https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-warm-pools.html)
```

Comment:
```suggestion
* `warm_pool` - (Optional) If this block is configured, add a [Warm Pool](https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-warm-pools.html)
```

Reply:
I noticed this one that I missed in the first review. I just committed it since it is very minor.

Everything looks great. Thanks for addressing all the feedback. I am going to run the acceptance tests and then I think this will b ready to merge.